mediator/med_enthalpy_mod.F90

```diff
@@ -0,0 +1,175 @@
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_hevap', hevap, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       allocate(hevap(nmax))
+    endif
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_hcond', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_hcond', hcond, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       allocate(hcond(nmax))
+    endif
+
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Faxa_snow', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Faxa_snow' , snow, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       call FB_GetFldPtr(is_local%wrap%FBImp(compatm,compatm), 'Faxa_snowc' , snowc, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+       call FB_GetFldPtr(is_local%wrap%FBImp(compatm,compatm), 'Faxa_snowl' , snowl, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+       allocate(snow(nmax))
+       snow = snowc + snowl
+    endif
+
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_hsnow', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_hsnow', hsnow, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       allocate(hsnow(nmax))
+    endif
+    
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_rofl', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_rofl' , rofl, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       call FB_GetFldPtr(is_local%wrap%FBImp(comprof, comprof), 'Forr_rofl', rofl, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    endif
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_hrofl', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_hrofl', hrofl, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       allocate(hrofl(nmax))
+    endif
+
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_rofi', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_rofi' , rofi, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       call FB_GetFldPtr(is_local%wrap%FBImp(comprof, comprof), 'Forr_rofi', rofi, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    endif
+    if(FB_fldchk(is_local%wrap%FBExp(compocn), 'Foxx_hrofi', rc)) then
+       call FB_GetFldPtr(is_local%wrap%FBExp(compocn), 'Foxx_hrofi', hrofi, rc=rc)
+       if (ChkErr(rc,__LINE__,u_FILE_u)) return
+    else
+       allocate(hrofi(nmax))
+    endif
+    do n = 1,nmax
+       ! Need max to ensure that will not have an enthalpy contribution if the water is below 0C
+       hrain(n)  = max((tocn(n) - tkfrz), 0._r8) * rain(n)  * cpfw
+       hsnow(n)  = min((tocn(n) - tkfrz), 0._r8) * snow(n)  * cpice
+       hevap(n)  = (tocn(n) - tkfrz) * min(evap(n), 0._r8)  * cpwv
+       hcond(n)  = (tocn(n) - tkfrz) * max(evap(n), 0._r8)  * cpwv
+       hrofl(n)  = max((tocn(n) - tkfrz), 0._r8) * rofl(n)  * cpsw
```

Comment:
Please change
`hrofl(n)  = max((tocn(n) - tkfrz), 0._r8) * rofl(n)  * cpsw
`
to 
`hrofl(n)  = max((tocn(n) - tkfrz), 0._r8) * rofl(n)  * cpfw`

Reply:
done